<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1246</width>
    <height>857</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>火箭计算系统</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1246</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>弹道计算</string>
    </property>
    <addaction name="ballAction"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="ballAction">
   <property name="text">
    <string>新建</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
